services/src/main/java/org/apache/druid/cli/CliCoordinator.java

```diff
@@ -272,10 +272,12 @@ public LoadQueueTaskMaster getLoadQueueTaskMaster(
         }
     );
 
-    modules.add(new LookupSerdeModule());
-
     if (beOverlord) {
       modules.addAll(new CliOverlord().getModules(false));
+    } else {
+      // Only add LookupSerdeModule if !beOverlord, since CliOverlord includes it, and having two copies causes
+      // the injector to get confused due to having multiple bindings for the same classes.
+      modules.add(new LookupSerdeModule());
```

Comment:
that is why `CliOverloard.getModules(boolean standalone)` takes the standalone flag also, so that we can take care of these things in there if/when needed.